src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -41,16 +42,16 @@ private static bool IsInvalidDouble(double number)
         /// <param name="returnBehavior">A flag that can be used to activate pre-defined early return behavior, such as returning Blank() if any argument is Blank().</param>
         /// <param name="targetFunction">The implementation of the builtin function.</param>
         /// <returns></returns>
-        private static FunctionPtr StandardErrorHandling<T>(
+        private static AsyncFunctionPtr StandardErrorHandling<T>(
```

Comment:
>AsyncFunctionPtr [](http://example.com/codeflow?start=23&length=16)

This is subtle but very significant. 

If I didn't produce async, then I'd need 2 copies of this file (one producing sync and another producing async). So to have 1 copy, everything becomes async.